src/test/kotlin/org/malachite/estella/offer/OfferServiceTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.malachite.estella.offer
+
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Test
+import org.malachite.estella.commons.models.offers.DesiredSkill
+import org.malachite.estella.commons.models.offers.Offer
+import org.malachite.estella.commons.models.offers.SkillLevel
+import org.malachite.estella.commons.models.people.HrPartner
+import org.malachite.estella.commons.models.people.Organization
+import org.malachite.estella.commons.models.people.User
+import org.malachite.estella.services.OfferService
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+import java.util.*
+import javax.sql.rowset.serial.SerialClob
+
+class OfferServiceTest {
+
+    private val repository = DummyOfferRepository()
+    private val offerService = OfferService(repository)
+
+    @AfterEach
+    fun setup() {
+        repository.clear()
+    }
+
+    @Test
+    fun `should add offer`() {
+        //when
+        offerService.addOffer(offer)
+        //then
+        expectThat(offerService.getOffer(offerId))
+            .isEqualTo(offer)
+    }
+
+    private val offerId = 1
+    private val organizationUUID = UUID.randomUUID()
+    private val user = User(null, "fname", "lname", "mail", "password")
+    private val offer = Offer(offerId, "name", SerialClob("descirption".toCharArray()),
```

Comment:
Skoro już mamy FakeLoadery to można by rozważyć by stamtąd brać dane i zapychać to testowe repository. Ale to sugestia na przyszłość

Reply:
Nie no, jak najbardziej sugestia na teraz. Już nie robimy Deadline Driven Development, więc wszystkie uwagi są ok XD

Reply:
Dobra, ogólnie jak w to popatrzyłem, to raczej nie pyknie - w FakeLoaderach masz przygotowane tak, żeby działało pod hibernate'a (nullowe id, itp). Ogólnie jeśli chcemy przetestować coś więcej niż dodawanie, musimy mieć dostęp do id, a szukać losowego, to nie na testy XD

Reply:
A to tak zgrubsza nie wystarczy dodać metody która będzie zamieniała ten null na index w liście ?

Reply:
@Davenury mógłbyś się odnieść do ostatniej propozycji?

Reply:
A sorry, mógłbym XD ogólnie wydaje mi się, że mogłoby wystarczyć, siądę po robocie i ogarnę

Reply:
Ogólnie dodałem - starałem się zrobić to jakoś ładnie, nie wiem, czy mi wyszło XD